.gitignore

```diff
@@ -16,6 +16,7 @@
 
 # build results
 /build
+/web/assets/asset.go
```

Comment:
aren't we pushing the assets?

Reply:
no we don't: https://github.com/src-d/gitbase-web/blob/master/.gitignore#L9

Reply:
wow... I had a reaaaaaly old version of `gitbase-web` in my PC, xD
so ashamed... :man_facepalming: